waiter/src/waiter/core.clj

```diff
@@ -943,19 +962,6 @@
                                    (let [{{:keys [query-state-fn]} :maintainer} router-state-maintainer
                                          service-gc-go-routine (partial service-gc-go-routine gc-state-reader-fn gc-state-writer-fn leader?-fn clock)]
                                      (scheduler/scheduler-broken-services-gc service-gc-go-routine query-state-fn scheduler scheduler-gc-config)))
-   :scheduler-maintainer (pc/fnk [[:routines service-id->service-description-fn]
```

Comment:
We no longer need a separate entry for the scheduler maintainer. This also ensures that no one is reading the scheduler syncer state.